tests/integration/test_einsum.py

```diff
@@ -261,6 +263,52 @@ def test_cast(expr, dtype):
     )
 
 
+@pytest.mark.parametrize(
+    "optimize",
+    [
+        (False),
+        ("optimal"),
+        ("greedy"),
```

Comment:
The parens around these values have no effect
```
In [1]: ("foo") == "foo"
Out[1]: True
```
I don't think you intended these to be 1-tuples, so the parens should just be removed